.rubocop.yml

```diff
@@ -26,6 +26,7 @@ Metrics/ClassLength:
     - lib/swagger_helpers.rb
 
 Metrics/MethodLength:
+  Max: 25
```

Comment:
I don't think it's a good idea to alter the rules globally without prior agreement between all the developers.

Personally, 25 lines for a method seem like too much, and only would make sense in certain specs or specific methods (and in those you could use an inline exception to the rule). Anyway, please ask @excelsior and @andersoncardoso whether or not they're ok with this.

Reply:
Like we talked yesterday, I agree with @aspino on this. I prefer a few 'disable' rules were needed and discourage big methods, than allowing then by default.